.gitignore

```diff
@@ -1 +1,2 @@
 tmp/
+.gh_access_token
```

Comment:
Is this necessary on the server because we need someones auth? This isn't a file I see in any repo locally, so mostly just curious.

Reply:
it might not be needed on the server.  figuring it all out.

Reply:
I also wonder, since this is a file I do not see locally (possibly something I've done wonky) if this would be better served in the readme (or in some doc) as something that may need to be added to your personal global .gitignore